### DISCUSSION TITLE
ESP_Buderus mit Homeassistant, MQTT, Node-Red, Influx-DB und Grafana

### DISCUSSION THREAD

**Discussion by djborizza** [dewenni/ESP_Buderus_KM271#126] [Q&A] [answered]
Hallo zusammen,

ich habe mittlerweile den ESP_Buderus bis hin zu Grafana implementiert. Funktioniert alles auch reibungslos bis auf 2 Sachen.
Seit dem ich MQTT eingeschaltet habe wird mir das Dashboard nicht mehr richtig angezeigt und die Einstellungen sind alle weg.
Es werden aber noch alle Daten geschrieben und mein Pushover geht auch noch:
![grafik](https://github.com/user-attachments/assets/1b6a22a5-a179-4e09-9a54-3775e8ee6c7e)
![grafik](https://github.com/user-attachments/assets/b4db3299-db8c-400e-80b8-06dd54c021ff)
![grafik](https://github.com/user-attachments/assets/ad7e4808-be79-4aa8-8045-002c36fb2ee8)

Wenn ich in Grafana die Dashboard Vorlage lade sind alle Werte leer:
![grafik](https://github.com/user-attachments/assets/915228ab-a51e-4b59-972b-c3eaf2574d12)

Beim Öffnen eines Wertes ist in der Querys kein Wert angegeben:
![grafik](https://github.com/user-attachments/assets/1de6de14-3c21-45b1-89d7-15d67495c432)

Sollte das nicht einfach alles 1:1 funktionieren da die Werte ja bei jedem gleich sind?

**Comment by dewenni** [comment]
Also das mit den fehlenden Werte in der WebUI ist vermutlich ein ähnliches Problem wie derzeit einige andere auch berichten. Warum das jetzt plötzlich so verstärkt auftritt ist mir zwar ein Rätsel, aber ich muss da ein bisschen was umbauen in der Kommunikation zwischen ESP und WebClient. Das ist derzeit einfach nicht stabil.
Gib mir noch ein paar Tage zeit dann gibt es ein neues Release wo das dann hoffentlich stabil für alle läuft.

Bezüglich Grafana würde ich nicht von einer "Plug and Play" Lösung sprechen.
Ich nutze z.B. InfluxDB 2.0 mit der Query Sprache "Flux"
Dein Screenshot sieht nach InfluxDB v1 aus.
Wenn du das auf V2 umstellst und auch die deutschen Namen für MQTT verwendest, dann sollte es aber schon größtenteils funktionieren.

**Reply by dewenni** [reply, marked as answer]
@djborizza ich habe es mal in die "Diskussionen" verschoben.

**Comment by djborizza** [comment]
Vielen Dank.
Das war es. Läuft nun prima.

Gruß Boris
